src/CommonITILTask.php

```diff
@@ -465,6 +465,16 @@ public function post_updateItem($history = true)
                         '_disablenotif' => true,
                     ];
                     $item->update($input2);
+                } elseif (
+                    $item->isStatusExists(CommonITILObject::ASSIGNED)
+                    && ($item->fields["status"] == CommonITILObject::PLANNED)
+                ) {
+                    $input2 = [
+                        'id'            => $item->getID(),
+                        'status'        => CommonITILObject::ASSIGNED,
```

Comment:
This seems a bit naive.
I don't have the code right now, but I think there is somewhere an example where the ticket return from an old status and compute the new one (assigned if any assignees, new if none)

Reply:
Is this better like that ? [24c6b28](https://github.com/glpi-project/glpi/pull/16895/commits/24c6b28749b491d4e9475010e0152cd0d46cd86e)